src/libs/zbxsysinfo/win32/net.c

```diff
@@ -24,12 +24,12 @@
 
 /* __stdcall calling convention is used for GetIfEntry2(). In order to declare a */
 /* pointer to GetIfEntry2() we have to expand NETIOPAPI_API macro manually since */
-/* part of it must be toghether with the pointer name in the parentheses.        */
+/* part of it must be together with the pointer name in the parentheses.        */
 typedef NETIO_STATUS (NETIOAPI_API_ *pGetIfEntry2_t)(PMIB_IF_ROW2 Row);
 
 /* GetIfEntry2() is available since Windows Vista and Windows Server 2008. In    */
 /* earlier Windows releases this pointer remains set to NULL and GetIfEntry() is */
-/* used dirrectly instead.                                                       */
+/* used directly instead.                                                       */
```

Comment:
I tried to fix all of the padding for `*/`, but it looks like i missed a few :-(